### DISCUSSION TITLE
9FUS won't update official firmware

### DISCUSSION THREAD

**Discussion by NightShift00** [roleoroleo/yi-hack-Allwinner#428] [Q&A] [answered]
My Yi 1080p has been having some issues. The recording playback on the timeline in the official app is scrambled. The videos are offset by like 7 hours and even then it jockeys back and forth between AM and PM. It's a mess. I'm assuming this is a symptom of leaving the base official firmware generations behind to maintain the hack. The stated official firmware on the camera in the app is 8.2.0. I'm trying to update it to 12.2.0 and then I'll apply the most up to date Allwinner hack to that, but no matter what I do I can't get 12.2.0 to stick. Does anyone know of a procedure to update in this situation?

**Comment by roleoroleo** [comment]
If you want to be sure, remove the hack with the unbrick procedure (using "hacked" option).
Then update the cam and apply again the hack.

**Reply by NightShift00** [reply, marked as answer]
Yeah, I was afraid of that. I'm sure I'll end up doing it eventually, but I shudder at the idea of that fine soldering. It looks like I might be good though. I've been having this issue since late December, but I just now got around to trying to address it again. In the time in-between there's been a hack update. It seems like the timeline issues were specific to y20ga 0.3.7, cause I updated to 0.3.8 and the timeline is playing nice again.

**Reply by roleoroleo** [reply]
Ok.
In any case, you don't need to use the iron soldering.
There is a soft procedure, check the wiki.
